tests/Feature/PrepareStreamsTest.php

```diff
@@ -27,7 +27,7 @@ public function it_groups_streams_by_date(): void
         $prepareStreamsAction = new PrepareStreams;
 
         // Act
-        $preparedStreams = $prepareStreamsAction->handle($streams, 'Europe/Vienna');
+        $preparedStreams = $prepareStreamsAction->handle($streams, 'UTC');
```

Comment:
the second param from this `handle` can also be removed since this method is not using a timezone anymore.